### PR TITLE
[SET-334] Harmonia jobs for installer and UMB notifier

### PR DESCRIPTION
**Pull request by spyrkob**
JIRA: https://projects.engineering.redhat.com/browse/SET-334

### CODE REVIEW

**Comment by spyrkob** [comment]
@rpelisse when you have a moment could you take a look at this PR? I'm sure there's a better way of doing a lot of this scripts ;)
For now this will only go into CCI branch, I'll worry about merging it with master after Olympus changes are done.

File: umb-build-message.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  --arg name "$RELEASE_NAME" \
+  --arg stream "$RELEASE_STREAM" \
+  --arg type "$RELEASE_TYPE" \
+  --arg version "$VERSION" \
+  --arg target-version "$BASE_VERSION" \
+  --arg eap-base-version "$BASE_VERSION" \
+  --arg url "$BUILD_URL" \
+  --argjson bits "$bits" )
+
+echo $msg | jq > message_body.json
```
Comment (rpelisse) [inline umb-build-message.sh:63]: "${msg}" ;)

**Review by rpelisse — CHANGES_REQUESTED** [review]
@spyrkob it's pretty solid stuff. i've seen a couple of very minor things (and I think you should pass shellcheck on the scripts, if you have not), but apart from that, it seems pretty solid.

Can you set a pr against Olympus branch too?

Thanks!

File: eap-installer.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+echo "Adding ${MAVEN_BIN_DIR} to PATH:${PATH}."
+export PATH=${MAVEN_BIN_DIR}:${PATH}
+
+if [ $# -ne 1 ]; then
+     echo "eap-installer needs action. Allowed values are [testsuite installer commons izpack]"
+     exit 2
+fi
+
+readonly ACTION=$1
+if [ "${ACTION}" != 'testsuite' ] && [ "${ACTION}" != 'installer' ] &&
```
[the remark this answers is not in the document]
Comment (spyrkob) [reply eap-installer.sh:21]: That would be a partial match though wouldn't it? So would match anything with those strings in like  "commons_build" or "not_installer"

[the remark this answers is not in the document]
Comment (rpelisse) [reply eap-installer.sh:21]: No you can make it "strict":
```
grep -e 'izpack'  | gre -e 'commons'  | grep -e 'testsuite'  | grep 'installer'
```

But honestly, it's not really much better, nor shorter, so you may forget about it :)

**Pull request merged**